### PR TITLE
Latest-Akhilesh Barrons Regression

### PR DESCRIPTION
**Pull request by AkhileshKhanduri25**
Updated the relevant Xpath, Element ID, query selector's values and span text values for Barron's regression page, Barron's Postback page, article page, watchlist page and Barron's mailbox page.

### CODE REVIEW

**Review by bhoomraogaripriyanka — COMMENTED** [review]
@AkhileshKhanduri25 ,Please find the comments and made changes accordingly

cc: @ArchanaSaini149

**Comment by AkhileshKhanduri25** [comment]
-> Fixed the Barron’s watchlist widget test case today and making changes to fix JSCreateListButton() path and Symbols to be searched when adding them on Watchlist under Barron’s Market data page and Barron’s Quotes page. 

-> Updated the code on the Line 117 on BarronsMailboxPage.robot

File: panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot:1]: working fine for me now

File: panda-qa/Resources/PageObjects/BarronsMarketDataPage.robot
Comment (bhoomraogaripriyanka) [inline panda-qa/Resources/PageObjects/BarronsMarketDataPage.robot:1]: Line 226 is working fine, 
Line 137 is not working for mw

**Review by bhoomraogaripriyanka — COMMENTED** [review]
Hi @AkhileshKhanduri25 ,Please find the comments on commits

cc: @ArchanaSaini149

**Review by ArchanaSaini149 — APPROVED** [review]

**Pull request merged**